packages/api-explorer/style/main.scss

```diff
@@ -565,3 +565,303 @@ form.rjsf fieldset {
 .string-value {
   word-break: break-all;
 }
+
+form.rjsf {
+  .field label:after {
+    display: none !important;
+  }
+  .form-group .form-group {
```

Comment:
Dupes?

Reply:
If I understand what you're asking, then no! This selector is for any `.form-group` that's a descendant of another `.form-group`.